src/extension.ts

```diff
@@ -77,6 +78,13 @@ export function activate(context: vscode.ExtensionContext) {
 
   MetadataViewerProvider.register(context);
 
+  MPQEditorProvider.register(context);
+
+  Logger.info("activated");
+
+  // Set context as a global
+  (global as any).extensionContext = context;
```

Comment:
Q. I wonder why is this needed?

Reply:
> Q. I wonder why is this needed?

AFAIK no. It's needed e.g. in tests to construct `MPQEditorProvider`, but it was used `constructor` only. So it can be omitted for now, because `constructor` is trivial.